test/test.js

```diff
@@ -441,6 +448,8 @@ test('.markdownlint.jsonc in cwd is used automatically', getCwdConfigFileTest('j
 
 test('.markdownlint.json in cwd is used automatically', getCwdConfigFileTest('json'));
 
+test('.markdownlint.toml in cwd is used automatically', getCwdConfigFileTest('toml'));
```

Comment:
This test should be removed if we don't introduce `.markdownlint.toml`.

Reply:
Oh, this test fails. Is that the problem you mentioned? Moot if we decide to remove it, though it'd be good to understand why.

Reply:
This is why it fails: https://github.com/BinaryMuse/toml-node/issues/55

Here's a commit to fix it that Codespaces won't let me push (you can apply it directly):

```diff
commit 0a1e1bdd6a58526db1bca0c4f02d54973b425d1d (HEAD -> add_toml_parser)
Author: David Anson <david@dlaa.me>
Date:   Mon Feb 19 23:06:57 2024 +0000

    Fix toml.parse issue of returning null-prototype objects.

diff --git a/markdownlint.js b/markdownlint.js
index 108add7..78be74c 100755
--- a/markdownlint.js
+++ b/markdownlint.js
@@ -37,7 +37,7 @@ function yamlParse(text) {
 }
 
 function tomlParse(text) {
-  return require('toml').parse(text);
+  return require('deep-extend')({}, require('toml').parse(text));
 }
 
 const exitCodes = {
```

Reply:
Please make the change above regardless. :)

Reply:
Thanks for the fix, would have taken me several days to figure that out :bow: .

Test is removed